ground/src/main/proto/com/google/ground/job.proto

```diff
@@ -19,11 +19,189 @@ syntax = "proto3";
+        // A task in which the user must capture their current device location.
+        CaptureLocation capture_location = 12;
+        // A task in which the user must take a photo.
+        TakePhoto take_photo = 13;
+    }
+
+    // A question whose response is entered manually as text.
+    message TextQuestion {
+        // Valid values for the `type` field.
+        enum Type {
+            UNSPECIFIED_TEXT_QUESTION_TYPE = 0;
+            // A single line of text.
+            SHORT_TEXT = 1;
+        }
+        // The type of text question. Defaults to SHORT_TEXT.
+        Type type = 1;
+    }
+
+    // A question whose response is entered manually as a number.
+    message NumberQuestion {
+        // Placeholder message. Fields may be added here in the future.
+    }
+
+    // A question whose response is entered manually as a data and/or time.
+    message DateTimeQuestion {
+        // Allowed values for the `type` field.
+        enum Type {
+            UNSPECIFIED_DATE_TIME_QUESTION_TYPE = 0;
+            DATE_ONLY = 1;
+            TIME_ONLY = 2;
+            BOTH_DATE_AND_TIME = 3;
+        }
+        // The type of data/time question.
+        Type type = 1;
+    }
+
+    // A question whose response is manually selected from a list of options.
+    message MultipleChoiceQuestion {
+        // Valid values for the `type` field.
+        enum Type {
+            UNSPECIFIED_MULTIPLE_CHOICE_QUESTION_TYPE = 0;
+            // Allow only one valid to be selected.
+            SELECT_ONE = 1;
+            // Allow one or more values to be selected.
+            SELECT_MULTIPLE = 2;
+        }
+        // The type of multiple choice question.
+        Type type = 1;
+        // The list of allowed multiple choice options.
+        repeated Option options = 2;
+
+        // A single selectable multiple choice option.
+        message Option {
+            // UUID of the option.
+            string id = 1;
+            // The index of the option when shown as a list.
+            int32 index = 2;
+            // The user-facing label of the option.
+            string label = 3;
+            // When true, the user may add qualifying text to the option. If
+            // label is omitted, clients should apply a default label
+            // "Other (specify)" or equivalent.
+            bool other_option = 4;
+        }
+    }
+
+    // A task in which the user must manually draw a geometry (point,
+    // polygon, etc.) on the map.
+    message DrawGeometry {
+        // Possible values for the `allowed_methods` field.
+        enum Method {
+            UNSPECIFIED_DRAW_GEOMETRY_METHOD = 0;
+            DROP_PIN = 1;
+            DRAW_AREA = 2;
+        }
+        // One or more methods which the user may use to complete the geometry
+        // task. If multiple methods are specified, data collectors are prompted
+        // which method should be used.
+        repeated Method allowed_methods = 1;
```

Comment:
I like this idea, we should implement it!

Reply:
Thanks! This is for #1744, which was filed to address the EUDR reporting of smaller plots, which the data collector can opt to report as a point rather than a polygon.